$NetBSD: patch-Makefile,v 1.3 2015/03/28 15:43:02 rhialto Exp $

Make prefix settable from the environment. The actual value will be set
via ${INSTALL_ENV} so the default is irrelevant.
Use the correct python binary.

--- Makefile.orig	2015-03-01 21:13:40.000000000 +0100
+++ Makefile	2015-03-28 16:23:00.000000000 +0100
@@ -5,7 +5,7 @@
 dist_dir := $(build_dir)/$(dist_name)
 
 DESTDIR :=
-prefix := /usr/local
+prefix ?= /usr/pkg
 
 ifeq ($(wildcard OpenGL),)
 	OpenGL_dir := "."
@@ -95,12 +95,12 @@
 
 install-program:
 ifeq ($(DESTDIR),)
-	python3 setup.py install	 --install-lib=$(prefix)/share/fs-uae-launcher	 --install-scripts=$(prefix)/share/fs-uae-launcher
+	${PYTHONBIN} setup.py install	 --install-lib=$(prefix)/share/fs-uae-launcher	 --install-scripts=$(prefix)/share/fs-uae-launcher
 	install -d $(DESTDIR)$(prefix)/bin
 	rm -f $(DESTDIR)$(prefix)/bin/fs-uae-launcher
 	ln -s ../share/fs-uae-launcher/fs-uae-launcher	 $(DESTDIR)$(prefix)/bin/fs-uae-launcher
 else
-	python3 setup.py install	 --root=$(DESTDIR)	 --install-lib=$(prefix)/share/fs-uae-launcher	 --install-scripts=$(prefix)/share/fs-uae-launcher
+	${PYTHONBIN} setup.py install	 --root=$(DESTDIR)	 --install-lib=$(prefix)/share/fs-uae-launcher	 --install-scripts=$(prefix)/share/fs-uae-launcher
 	install -d $(DESTDIR)$(prefix)/bin
 	rm -f $(DESTDIR)$(prefix)/bin/fs-uae-launcher
 	ln -s ../share/fs-uae-launcher/fs-uae-launcher	 $(DESTDIR)$(prefix)/bin/fs-uae-launcher
